src/main/webapp/pluggable/dataset-analisys-results.xhtml

```diff
@@ -0,0 +1,94 @@
+                        <h:outputText value="#{artifact.getName()}" />
+                    </p:commandLink>
+                </p:column>
+            </p:dataTable>
+            
+        </h:panelGroup>
+    </ui:fragment>
+
+    <ui:fragment rendered="#{!datasetAnalysisTab.showAnalysisResults()}">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                <div class="col-sm-9">
+                    <ui:fragment rendered="#{datasetAnalysisTab.isDatasetInDraft()}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.dataset.in.draft.message']}" />
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{datasetAnalysisTab.isDatasetUnderEmbargo() and !datasetAnalysisTab.isPermissionToViewFiles()}">
+                        <p>
+                            <h:outputFormat value="#{mxrdrBundle['dataset.analysisTab.embargo.message']}">
+                                <f:param value="#{datasetAnalysisTab.getEmbargoDateForDisplay()}"/>
+                            </h:outputFormat>
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{!datasetAnalysisTab.isDatasetInDraft() and
+                                              (!datasetAnalysisTab.isDatasetUnderEmbargo() or
+                                              (datasetAnalysisTab.isDatasetUnderEmbargo() and datasetAnalysisTab.isPermissionToViewFiles()) and
+                                              datasetAnalysisTab.isAnalysisInProgress())}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.analysis.in.progress.message']}" />
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{!datasetAnalysisTab.isDatasetInDraft() and
+                                              (!datasetAnalysisTab.isDatasetUnderEmbargo() or
+                                              (datasetAnalysisTab.isDatasetUnderEmbargo() and datasetAnalysisTab.isPermissionToViewFiles()) and
+                                              datasetAnalysisTab.isAnalysisNotPerformed())}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.analysis.not.performed.message']}" />
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{!datasetAnalysisTab.isDatasetInDraft() and
+                                              (!datasetAnalysisTab.isDatasetUnderEmbargo() or
+                                              (datasetAnalysisTab.isDatasetUnderEmbargo() and datasetAnalysisTab.isPermissionToViewFiles()) and
+                                              datasetAnalysisTab.isAnalysisFailure())}">
```

Comment:
I would avoid having complicated rendered conditions - it easily can lead to unpredicted results in case of some further changes. Instead of this I recommend to return reason from backing bean why we can't show the results.
And in xhtml just checking this one variable. It could be done by comparing strings or even better by comparing enum values (see: `<o:importConstants />`)

Reply:
All conditions checks moved to bean